src/manage_sql_nvts.c

```diff
@@ -1558,13 +1636,20 @@ manage_update_nvt_cache_osp (const gchar *update_socket)
                    "                 0);")
           == 0)
         {
-          check_preference_names ("config_preferences");
-          check_preference_names ("config_preferences_trash");
+          check_old_preference_names ("config_preferences");
+          check_old_preference_names ("config_preferences_trash");
+
+          /* Force update of names in new format in case hard-coded names
+           *  in migrator are outdated */
```

Comment:
```suggestion
           * in migrator are outdated */
```

Reply:
I've rewritten this line in 991b9d11dfa93cde1a6eaa31aec557446570e3a3.